### PR TITLE
Update alert-controller.ts

### PR DESCRIPTION
**Pull request by Ritzlgrmft**
#### Short description of what this resolves:
Default value for `enableBackdropDismiss` documented.

#### Changes proposed in this pull request:
n/a

**Ionic Version**: 1.x / 2.x / 3.x
3.x

**Fixes**: #
n/a

### CODE REVIEW

File: src/components/alert/alert-controller.ts [hunk 1]
```diff
@@ -136,7 +136,7 @@ import { Config } from '../../config/config';
  *  | cssClass              | `string`  | Additional classes for custom styles, separated by spaces.                |
  *  | inputs                | `array`   | An array of inputs for the alert. See input options.                      |
  *  | buttons               | `array`   | An array of buttons for the alert. See buttons options.                   |
- *  | enableBackdropDismiss | `boolean` | Whether the alert should be dismissed by tapping the backdrop.            |
+ *  | enableBackdropDismiss | `boolean` | Whether the alert should be dismissed by tapping the backdrop. Default true. |
```
Comment (AmitMY) [inline src/components/alert/alert-controller.ts]: Style: Align all `|` to be in the same column.

**Comment by Ritzlgrmft** [comment]
table aligned

**Comment by manucorporat** [comment]
cc @mhartington

**Comment by mhartington** [comment]
Thanks @Ritzlgrmft !

**Pull request closed without merging**